diag_manager/include/fms_diag_reduction_methods.inc

```diff
@@ -201,4 +201,71 @@ subroutine DO_TIME_MAX_ (data_out, data_in, mask, is_masked, bounds_in, bounds_o
       enddo
     enddo
   endif
-end subroutine DO_TIME_MAX_
\ No newline at end of file
+end subroutine DO_TIME_MAX_
+
+!> Update the output buffer and counter for the time_sum reduction
+!! Updates elements of the running field output buffer (data_out)
+!! For time_sum, just sums passed field data into the output buffer
+subroutine DO_TIME_SUM_(data_out, data_in, mask, is_masked, bounds_in, bounds_out, missing_value )
+  real(FMS_TRM_KIND_),       intent(inout) :: data_out(:,:,:,:,:) !< output data
+  real(FMS_TRM_KIND_),       intent(in)    :: data_in(:,:,:,:)    !< data to update the buffer with
+  logical,                   intent(in)    :: mask(:,:,:,:)       !< mask
+  logical,                   intent(in)    :: is_masked           !< .True. if the field is using a mask
+  type(fmsDiagIbounds_type), intent(in)    :: bounds_in           !< indices indicating the correct portion
+                                                                  !! of the input buffer
+  type(fmsDiagIbounds_type), intent(in)    :: bounds_out          !< indices indicating the correct portion
+                                                                  !! of the output buffer
+  real(FMS_TRM_KIND_),       intent(in)    :: missing_value       !< Missing_value for data points that are masked
```

Comment:
Isn't this a fill value?

Reply:
The _FillValue and a missing_value for variables are both set to the same number when i look at the files produced, i just used the argument name from the other reductions.